Weather_App/Weather_App/Data/Repository/LocationRepository.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  LocationRepository.swift
+//  Weather_App
+//
+//  Created by ho.bao.an on 14/05/2024.
+//
+
+import Foundation
+import MapKit
+import RxSwift
+
+final class LocationSearchRepository {
+    static func getSearchResults(for searchText: String) -> Observable<MKMapItem> {
```

Comment:
đoạn này sao lại phải dùng static vậy em ?

Reply:
Dạ em dùng static ở đây để khi em gọi hàm getSearchResults(for:) ở nơi khác mà không cần tạo instance ạ!
Ví dụ return LocationSearchRepository.getSearchResults(for: searchText) ạ!

Reply:
@anhb-3436 
làm như này là vi phạm dependency injection nhé, em sẽ ko thể viết test cho `LocationGateway` của em :)))